sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -569,8 +570,10 @@ abstract class ExternalCatalogSuite extends SparkFunSuite {
         // then be caught and converted to a RuntimeException with a descriptive message.
         case ex: RuntimeException if ex.getMessage.contains("MetaException") =>
           throw new AnalysisException(
-            errorClass = "_LEGACY_ERROR_TEMP_3066",
-            messageParameters = Map("msg" -> ex.getMessage))
+            errorClass = "_LEGACY_ERROR_TEMP_2193",
```

Comment:
Refer to this:
https://github.com/apache/spark/blob/b106f80521d609b280950f63ff670c1f39ec3cee/sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala#L383-L384

https://github.com/apache/spark/blob/b106f80521d609b280950f63ff670c1f39ec3cee/sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala#L1644-L1651